4week/src/components/Navbar.tsx

```
@@ -2,21 +2,27 @@
-import { QUERY_KEY } from "../constants/key"; // QUERY_KEY 사용
+import { QUERY_KEY } from "../constants/key";
+import { Menu } from "lucide-react"; // 햄버거 메뉴 아이콘 임포트
 
-export default function Navbar() {
+// HomeLayout에서 프롭스를 받아옵니다.
+interface NavbarProps {
+  isSidebarOpen: boolean;
+  setIsSidebarOpen: () => void;
+}
+
+export default function Navbar({ isSidebarOpen, setIsSidebarOpen }: NavbarProps) {
```

ESLint

[error] 'isSidebarOpen' is defined but never used.

(@typescript-eslint/no-unused-vars)